contract/users_response.go

```diff
@@ -0,0 +1,28 @@
+package contract
+
+import (
+	"time"
+
+	"github.com/vkhichar/asset-management/domain"
+)
+
+type User struct {
+	ID        int       `json:"id"`
+	Name      string    `json:"name"`
+	Email     string    `json:"email"`
+	IsAdmin   bool      `json:"is_admin"`
+	CreatedAt time.Time `json:"created_at"`
+	UpdatedAt time.Time `json:"updated_at"`
+}
+
+func DomainToContract(d *domain.User) User {
+	var u User
```

Comment:
you can also define it in this way, u := User{
ID : d.ID
} add all fields. As this is more readable.